Python/marshal.c

```diff
@@ -1534,6 +1542,33 @@ r_object(RFILE *p)
         retval = Py_NewRef(v);
         break;
 
+    case TYPE_SLICE:
+    {
+        int idx = r_ref_reserve(flag, p);
```

Comment:
Actions is complaining about the conversion here
```suggestion
        Py_ssize_t idx = r_ref_reserve(flag, p);
```